info/v1/container.go

```diff
@@ -516,6 +516,31 @@ type FsStats struct {
 	WeightedIoTime uint64 `json:"weighted_io_time"`
 }
 
+// NvidiaGpuStats are measured using NVML.
+type NvidiaGpuStats struct {
+	// Minor number of the GPU device (/dev/nvidia[minor_number])
+	MinorNumber uint64 `json:"minor_number"`
```

Comment:
How do we plan to identify a GPU in the device plugin model?  It would be nice if the minimum set of identifying information was the same across cAdivsor and kubernetes (e.g. maker, model, minor_number)?

Reply:
Good point. It depends on the device plugin implementation (which is outside the kubernetes/kubernetes tree). In the current implementation, when you ask for `nvidia.com/gpu` resource, the GPU IDs are `nvidia0`, `nvidia1` etc. So, just the minor numbers. I think there are plans to request different models of GPUs through annotations possibly but nothing concrete has happened in that regard yet.

cc - @jiayingz